src/argilla/training/transformers.py

```diff
@@ -258,6 +251,8 @@ def compute_metrics_text_classification(eval_pred):
                 func = compute_metrics_text_classification
 
         elif self._record_class == rg.TokenClassificationRecord:
+            import seqeval as _  # noqa
```

Comment:
Is this just to trigger an ImportError if it doesn't exist? I think loading `"seqeval"` with evaluate throws an ImportError as well, with some additional context. 
Perhaps we can use `require_dependency` or however that function was called here?